### PR TITLE
make new typo3 composer mode work

### PR DESCRIPTION
**Pull request by zdreicom**

### CODE REVIEW

**Comment by zdreicom** [comment]
@haassie i just fix it on my own

**Comment by zdreicom** [comment]
@RinyVT you are right, works fine for me too just change my pl

File: Classes/Utility/PathUtility.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+namespace YoastSeoForTypo3\YoastSeo\Utility;
+
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use TYPO3\CMS\Core\Utility\PathUtility as TYPO3PathUtility;
+
+class PathUtility
+{
+    public static function getPublicPathToResources(): string
+    {
+        return TYPO3PathUtility::getAbsoluteWebPath(ExtensionManagementUtility::extPath('yoast_seo')) . 'Resources/Public';
```
Comment (RinyVT) [inline Classes/Utility/PathUtility.php:12]: Looks like the `Resources/Public` is still outside of the `getAbsoluteWebPath` and thus will not be replaced by core?

**Review by RinyVT — CHANGES_REQUESTED** [review]

File: Classes/Utility/PathUtility.php [hunk 2]
```diff
@@ -9,6 +9,6 @@ class PathUtility
 {
     public static function getPublicPathToResources(): string
     {
-        return TYPO3PathUtility::getAbsoluteWebPath(ExtensionManagementUtility::extPath('yoast_seo')) . 'Resources/Public';
+        return TYPO3PathUtility::getAbsoluteWebPath(ExtensionManagementUtility::extPath('yoast_seo') . 'Resources/Public');
```
Comment (zdreicom) [inline Classes/Utility/PathUtility.php:12]: @RinyVT i just check TYPO3 comments  for this method "     * Until we have a replacement for this API, the safest way to call this method is by providing absolute filesystem paths
     * and use \TYPO3\CMS\Core\Utility\PathUtility::getPublicResourceWebPath whenever possible."

I do already have a change using "getPublicResourceWebPath" witch is testst in new and old composer mode i will commit it shordly

**Pull request merged**